src/migrations/20221103000000-emergent-goals-not-rttapa.js

```diff
@@ -0,0 +1,82 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(
+      async (transaction) => {
+        const loggedUser = '0';
+        const sessionSig = __filename;
+        const auditDescriptor = 'RUN MIGRATIONS';
+        await queryInterface.sequelize.query(
+          `SELECT
+            set_config('audit.loggedUser', '${loggedUser}', TRUE) as "loggedUser",
+            set_config('audit.transactionId', NULL, TRUE) as "transactionId",
+            set_config('audit.sessionSig', '${sessionSig}', TRUE) as "sessionSig",
+            set_config('audit.auditDescriptor', '${auditDescriptor}', TRUE) as "auditDescriptor";`,
+          { transaction },
+        );
+
+        await queryInterface.sequelize.query(
+          `
+          SELECT "ZAFSetTriggerState"(null, null, null, 'DISABLE');
+          `,
+          { transaction },
+        );
+
+        // remove old column from users table
+        await queryInterface.sequelize.query(
+          `UPDATE "Goals" g
+          SET "isRttapa" = false
+          WHERE COALEASE(g."isFromSmartsheetTtaPlan",false) = false;`,
```

Comment:
isRttapa is an enum of "Yes", "No", not a bool

It's this way as a convenience for the way the UI consumes it - I mean, an enum instead of a boolean because, in the mockups, the field was not indicated as required (or at least that's how I remember it, that could be my mistake), and so the intention seemed to be it could be null, and in that case it makes checking false a little more complicated. 

I'm not sure if the designers were included on this decision (maybe it was their idea, I don't know) but if not, I would confirm that we want to auto set all non-RTTAPA existing goals to emergent and confirm that the field shouldn't be required with either them or Jenn.

Reply:
OK, I will have Jenn confirm before we merge. I have updated to use the enum.